data/src/main/java/com/globant/data/service/response/PokemonCardResponse.kt

```diff
@@ -6,13 +6,13 @@ class PokemonCardResponse(
     var nationalPokedexNumber: Int?,
     var imageUrl: String,
     var types: List<String>?,
-    var supertype: String?,
-    var subtype: String?,
+    var supertype: String,
+    var subtype: String,
     var evolvesFrom: String?,
-    var hp: String,
+    var hp: String?,
```

Comment:
are these still necessary?

Reply:
yes, because some cards doesn't have all the attributes, so it can be null